server/lib/oauth-model.ts

```diff
@@ -75,7 +75,13 @@ function getRefreshToken (refreshToken: string) {
 async function getUser (usernameOrEmail: string, password: string) {
   logger.debug('Getting User (username/email: ' + usernameOrEmail + ', password: ******).')
 
-  const user = await UserModel.loadByUsernameOrEmail(usernameOrEmail)
+  let user : UserModel
+  if (CONFIG.AUTH.LDAP.ENABLED) {
+    user = await UserModel.findOrCreateLDAPUser(usernameOrEmail)
+  }
+  if (!user && CONFIG.AUTH.LOCAL.ENABLED) {
```

Comment:
I think we should put the local check first. Because users that were created before the LDAP activation should be able to log in with their "old" account.

Reply:
Usually it's not the good strategy : 
1) If LDAP is enabled, you first  check if login and password are ok in LDAP server.
2) If not, and if fallback login option is activated, you check in local app database if login/password are OK.

Usually LDAP server is the authentification security reference, so it's normal that's if activated it's checked first.